### DISCUSSION TITLE
Unable to remove reviewers while creating a pull request

### DISCUSSION THREAD

**Discussion by doxman** [community/community#109758] [Pull Requests]
### Select Topic Area

Bug

### Body

I'm seeing a similar issue to https://github.com/orgs/community/discussions/69208 ; when I de-select reviewers from the selector, they don't actually get removed.

This is distinct though, because it only happens while creating a PR (once it's actually created, I can remove the reviewers). So the problem here is less severe, but still worth addressing (ex. I accidentally select someone I didn't want as a reviewer, and now that person will be bothered with emails about my PR)

# Reproduction steps
Start creating a pull request from any branch.

Add one or more reviewers.

Try to remove a reviewer - the selector lets me check/uncheck, but the name never leaves the list, and is still on the PR when I save it.

# Fault Type
Permanent - Occurring repeatably

# Blocking?
No

**Comment by InsertCreativityHere** [comment]
Can confirm that this issue is still present.
It's been happening to me (Firefox 130 on Windows 11) for at least a year.
But I only just now decided to see if there was an open issue for it.

Usually I can get around this by refreshing the page, then adding another random reviewer,
waiting a couple seconds,
then after I remove the random reviewer, I will also be able to remove the stuck reviewer.

**Comment by cbatson** [comment]
Still occurring April 2026. The above workaround does not work for the repo I'm in. It says "Request up to 1 reviewer" so I cannot select more than one reviewer, nor can I deselect a reviewer once chosen. However, closing the browser and starting over with "Compare & pull request" on the branch allowed it to "forget" the reviewer I'd accidentally selected.

**Comment by chrisdel101** [comment]
How do I deselect a reviewer? I tried everything obvious
Seems like it should be super simple but I can't figure it out.

EDIT: I fig'd out this over-engineered approach which [removes the review request](https://github.com/orgs/community/discussions/23054#discussioncomment-17194467)

**Comment by ryanward2453-dotcom** [comment]
This looks like a state sync bug during PR creation where deselected reviewers aren’t being persisted correctly. It’s similar to how in workflows like [merchant services lead generation](https://shorturl.at/YdK0D), incorrect selections early in the flow can trigger unwanted follow-ups.
